README.md

```diff
@@ -172,40 +172,60 @@ quality_report(folder_sequences = directory_sequences,
 ```
 ## Default parameters:
 
-The default parameters were derived from data sets analyzing B cell receptor sequences from rhesus macaques, macaca mulatta. These defaults can be used in other experiments or can be changed dependent on the users' needs. Below are potential defaults for B cell receptors for macaques and T cell receptors for mice. For other species not listed or if you are uncertain, a `trimmed_mean_quality` or minimum phred quality score = 30 should be used as this returns an average of 99.9% base call accuracy for the analyzed samples.    
+The default parameters were derived from a data set analyzing B cell receptor sequences from rhesus macaques (_Macaca_ _mulatta_) generated by Ols et al., _Immunity_ 2023. To generate this dataset, primers from Sundling et al., _Sci._ _Transl._ _Med_ 2013 and _J._ _Immunol._ _Methods_ 2013 were used. Those primers were developed to capture the entire V region from heavy and light chain from macaque B cell receptors, the mean length of those sequences was 460 basepairs (bp). The default parameters were also tested for a human B cell receptor dataset generated using primers from Doria-Rose et al., _J._ _Virol._ 2016. Additionally, we have tested scifer for gamma delta T cell receptor sequences from human and mice (_Mus_ _musculus_), for those it was required to change the default parameters due to its shorter amplicon length of 250 bp based on the primers used. The most important information is the length of your expected amplicon, with a different amplicon length you should change the default parameters, most importantly the `raw_length` and `trim_finish` parameters since they rely on your amplicon length. The default of minimum mean Phred quality score equal to 30 is standard to achieve mean of 99.9% base call accuracy. Below you can find the defaults that are used by scifer and what you should change for other species or amplicons. These defaults can be used in other experiments or can be changed dependent on the users' needs. Below are potential defaults for B cell receptors for macaques and T cell receptors for mice.     
 
-Default parameters for B cell receptors in macaques: 
+Default parameters for B cell receptors in macaques and humans as meantioned above: 
 
 ```r
-# `raw_length` = 400 
+# `raw_length` = 343 # based on your expected amplicon length
 # `trim_start` = 50 
```

Comment:
add similar comments to all the parameters

Reply:
Addressed in commit 6bd97d120db2f1209e72cd1ca00641db1dc7e078